frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/ClusterDestroy.tsx

```diff
@@ -45,11 +45,25 @@ export function ClusterDestroy(props: { isLoading: boolean; cluster: Cluster })
       isLoading={props.isLoading}
       loadingTitle={loadingTitle}
       loadingMessage={
-        <Trans
-          i18nKey={`${props.cluster?.status}.inprogress.message`}
-          values={{ clusterName: props.cluster?.displayName }}
-          components={{ bold: <strong /> }}
-        />
+        props.cluster?.status === ClusterStatus.destroying && isHybrid ? (
```

Comment:
nit: The only thing changing here is the `i18nKey`. What about calculating it a few lines above before the `return` statement?

Reply:
will do!

Reply:
fixed